packages/vite/src/node/index.ts

```diff
@@ -7,6 +7,7 @@ export {
   loadConfigFromFile,
   resolveConfig,
   sortUserPlugins,
+  configDefaults,
```

Comment:
I noticed that this only exposes the value for ESM entry point and does not expose the value for CJS entry point. Are we fine with that? To make that work, `configDefaults` and the values used in it needs to be moved to `constants.ts` and makes a bit difficult to read the code.

Reply:
Yeah I think we'd have to expose it in CJS for consistency unfortunately, otherwise they could get a confusing error of it not working.